apps/mark/backend/src/server.test.ts

```diff
@@ -1,35 +1,32 @@
+import { expect, test, vi } from 'vitest';
 import { mockBaseLogger } from '@votingworks/logging';
 import tmp from 'tmp';
 import { buildMockInsertedSmartCardAuth } from '@votingworks/auth';
-import { mockOf } from '@votingworks/test-utils';
 import { initializeSystemAudio } from '@votingworks/backend';
-import { PORT } from './globals';
 import { start } from './server';
 import { createWorkspace } from './util/workspace';
 
-jest.mock(
-  '@votingworks/backend',
-  (): typeof import('@votingworks/backend') => ({
-    ...jest.requireActual('@votingworks/backend'),
-    initializeSystemAudio: jest.fn(),
-  })
-);
+vi.mock(import('@votingworks/backend'), async (importActual) => ({
+  ...(await importActual()),
+  initializeSystemAudio: vi.fn(),
+}));
 
 test('can start server', async () => {
-  const auth = buildMockInsertedSmartCardAuth();
-  const baseLogger = mockBaseLogger({ fn: jest.fn });
+  const auth = buildMockInsertedSmartCardAuth(vi.fn);
+  const baseLogger = mockBaseLogger({ fn: vi.fn });
   const workspace = createWorkspace(
     tmp.dirSync().name,
-    mockBaseLogger({ fn: jest.fn })
+    mockBaseLogger({ fn: vi.fn })
   );
 
   const server = await start({
     auth,
     baseLogger,
-    port: PORT,
+    // pick an available port
+    port: 0,
```

Comment:
This was unrelated to the vitest change, but without this the tests would fail if port 3002 was in use.